src/app/app.module.ts

```diff
@@ -19,7 +19,8 @@ import { PasswordResetComponent } from './pages/password-reset/password-reset.co
 import { EnterResetPasswordComponent } from './pages/enter-reset-password/enter-reset-password.component';
 import { AuthGuard } from './guards/auth.guard';
 import { NoAuthGuard } from './guards/no-auth.guard';
-
+import { InternationalPaymentStatusComponent } from './pages/payment/international-payment-status/international-payment-status.component';
+import { PaymentModule } from './pages/payment/payment.module';
 
```

Comment:
*InternationalPaymentStatusComponent* is in the app module because for that particular component I did not want to have the header and footer as they would appear twice on the particular page where the link is called